Makefile

```diff
@@ -60,3 +61,13 @@ clean:
 	rm -rf deployments/dist
 	find . -type d -name __pycache__ -exec rm -rf {} +
 	find . -type d -name .mypy_cache -exec rm -rf {} +
+
+install:
+	yarn install
+	poetry install
+ifeq ($(ARCH),arm64)
+	$(eval ATOMICS := $(shell poetry export | egrep 'atomics==' | awk '{print $$1}'))
```

Comment:
Nitpick: `egrep` is deprecated (not sure about the version you have on Mac, though)

```
egrep: warning: egrep is obsolescent; using grep -E
```

Reply:
you're right. used `grep -E` in difflint work for that reason.